cpp/json_schema_converter.cc

```diff
@@ -661,12 +779,38 @@ std::string JSONSchemaConverter::VisitInteger(
       schema,
       {
           "multipleOf",
-          "minimum",
-          "maximum",
-          "exclusiveMinimum",
-          "exclusiveMaximum",
       }
   );
+  std::string range_regex = "";
+  try {
+    if (schema.count("minimum") || schema.count("maximum") || schema.count("exclusiveMinimum") ||
+        schema.count("exclusiveMaximum")) {
+      std::optional<int> start, end;
+      if (schema.count("minimum")) {
+        double start_double = schema.at("minimum").get<double>();
+        start = static_cast<int>(start_double);
+      }
+      if (schema.count("exclusiveMinimum")) {
+        double start_double = schema.at("exclusiveMinimum").get<double>();
+        start = static_cast<int>(start_double);
+      }
+      if (schema.count("maximum")) {
+        double end_double = schema.at("maximum").get<double>();
+        end = static_cast<int>(end_double);
+      }
+      if (schema.count("exclusiveMaximum")) {
+        double end_double = schema.at("exclusiveMaximum").get<double>();
+        end = static_cast<int>(end_double);
+      }
+      range_regex = generateRangeRegex(start, end);
+    }
+    if (!range_regex.empty()) {
+      std::string converted_regex = RegexToEBNF(range_regex, false);
+      return converted_regex;  // not " " for numbers
+    }
+  } catch (const std::exception& e) {
```

Comment:
Please remove the usage of try-catch block and std::exception. The reason is we maintain a typescript binding running in WASM, which has not supported exceptions yet. Just throw an error at the error line.